ISISScientificComputing/autoreduce
# 486 submitted by on job queue page

### Summary of work

- Refactored the code in views.py to have a method that looks up user/team names based on user ID to be used for assigning who submitted a run.
-Changed run_summary() and run_queue() in views.py to use this new method.
-Changed run_queue.html to correctly display the 'submitted by' information on the Run Queue page.
-Created new release notes.

### How to test your work
- Set your credentials.ini to point to the development environment start the webapp
- Navigate to (<webapp_address>/runs/GEM/90107/1/) and ensure that the user is stated as Started by: Development team
- Navigate to (<webapp_address>/runs/GEM/90107/) and ensure that the user is stated as Started by: Autoreduction service
- Navigate to (<webapp_address>/runs/GEM/90107/3/) and ensure that the user is stated as Started by: Elliot Oram
-Navigate to (<webapp_address>/runs/queue/) and ensure the 'Submitted By' columns is correctly populated.
### Additional comments
The way to iterate over more than one object simultaneously with Django is to zip the objects into the context dictionary.

Fixes #486
